distribution/client/src/lib/downloader.js

```diff
@@ -27,20 +27,38 @@ class Downloader {
 
     logger.info('Fetching %s and saving to %s', item, filename);
 
-    return new Promise((resolve, reject) => {
-      fetch(item).then((res) => {
-        const output = fs.createWriteStream(filename);
+    let options = {
+      uri: item,
+      verboseLogging: true,
+      method: 'GET',
+      headers: {
+        'User-Agent': 'evergreen-client'
```

Comment:
Should probably mention Jenkins somewhere, no?

Reply:
Good question. I actually almost wrote something like "Jenkins-Essentials", and went back to evergreen-client, following somehow https://github.com/jenkinsci/jep/tree/master/jep/301#evergreen-client. 
I expect that, anyway, this should always be hitting Jenkins Project's related infra, so it should always be obvious what it's referring to, but well. Likely `jenkins-evergreen-client` would make it crystal clear indeed.

Will change it to that tomorrow, I think. No strong opinion.

Any opinion on this @rtyler.

Reply:
There's no need to make the user agent more explicit